app/javascript/components/explore/GenomeView.js

```diff
@@ -32,6 +33,11 @@ function GenomeView({ studyAccession, bamFileName, isVisible, updateExploreParam
   // re-render IGV any time the listing of bamFiles changes
   useEffect(() => {
     if (bamFileList && bamFileList.bamAndBaiFiles.length && isVisible) {
+      // show profile warning from non-existent token due to incomplete Terra registration
+      if (window.accessToken === '') {
+        $('#' + igvContainerId).parent().append(profileWarning)
```

Comment:
There's probably a more React-idiomatic way to do this, but I'm fine with it as reads clearIy with no obvious functional problem.

Reply:
79723b0